admin_guide/preventing_node_placement.adoc

```diff
@@ -0,0 +1,119 @@
+====
+This process involves the *pod/binding* permission
+link:../admin_guide/manage_authorization_policy.html[role], which is needed to
+target particular nodes. The constraint on the use of the `nodeSelector` field
+of a pod configuration is based on the *pods/binding* permission and the
+`nodeSelectorLabelBlacklist` configuration option.
+====
+
+== Constraining Pod Placement
+
+The `nodeSelectorLabelBlacklist` field of a master configuration file gives you
+control over the labels that certain roles can specify in new
+`nodeSelector` fields of a pod configuration. Users, service accounts, and
+groups that have the *pods/binding* permission can specify any node selector.
+Those without the *pods/binding* permission are prohibited from setting a
+`nodeSelector` for any label that appears in `nodeSelectorLabelBlacklist`.
+
+For example, you might have a cluster consisting of five data centers spread
+across two regions. In the US, "us-east", "us-central", and "us-west"; and
+in APAC, "apac-east" and "apac-west". Each node in each geographical region
+link:../admin_guide/manage_nodes.html#updating-labels-on-nodes[should be labeled
+accordingly]. For example, `region: us-east`.
+
+As a cluster administrator, you can create an infrastructure where application
+developers are deploying pods onto the nodes closest to their geographical
+location. You can
+link:../admin_guide/managing_projects.html#using-node-selectors[create a node
+selector], grouping the US data centers into `superregion: us` and the APAC
+data centers into `superregion: apac`.
+
+To maintain an even loading of resources per data center, you can add the
+specified `region` label to the `nodeSelectorLabelBlacklist` section of a master
+configuration. Then, whenever a developer located in the US creates a
+pod, it is deployed onto a node in one of the regions with the `superregion: us`
+label. If the developer tries to target a specific region for their pod, they
+will receive an error. If they try again, without the node selector on
+their pod, it can still be deployed onto the region they tried to target,
+because `superregion: us` is set as the project-level node selector, and those
+nodes are labeled with `superregion: us`.
+
+To prevent users from targeting specific nodes, modify the OpenShift master
+configuration, and the Kubernetes resource configuration.
+
+. Ensure you have the desired
+link:../admin_guide/manage_nodes.html#updating-labels-on-nodes[labels] and
+link:../admin_guide/managing_projects.html#using-node-selectors[node selector]
+set up in your environment.
+
+. Modify the master configuration file
+(*_/etc/origin/master/master-config.yaml_*) in two places:
++
+.. First, the `admissionConfig` section:
++
+====
+----
+...
+admissionConfig
+  pluginConfig
+    PodNodeConstraints
+      configuration
+        apiversion: v1
+        kind: PodNodeConstraintsConfig
+        nodeSelectorLabelBlacklist:
+          - kubernetes.io/hostname
+          - <label_1>
```

Comment:
@bfallonf
Sorry, i was not specific.  The map keys requiring a terminating colon are:

- `admissionConfig`
- `pluginConfig`
- `PodNodeConstraints`
- `configuration`

For a quick read on fundamental YAML syntax, see also [YAML in a Nutshell](https://access.redhat.com/documentation/en/red-hat-enterprise-linux-atomic-host/7/getting-started-with-containers/chapter-13-yaml-in-a-nutshell) (one of the first writings i did, so as yet unhindered by normal enterprise-y title humbuggery... :smile:).